ang/civicase/case/search/directives/search.directive.js

```diff
@@ -181,6 +210,42 @@
       return des;
     }
 
+    /**
+     * Watches changes to the case role filters, prepares the params to be sent
+     * to the API and appends them to the filters object.
+     */
+    function caseRoleWatcher () {
+      var filters = $scope.filters;
```

Comment:
Is it worth it to save it in a new variable?

Reply:
Looks a bit cleaner. But I can remove it if you think it's not worth it.